src/test/java/com/rocketden/main/socket/GameSocketTests.java

```diff
@@ -47,25 +51,23 @@
     private TestRestTemplate template;
 
     private static final String CONNECT_ENDPOINT = "ws://localhost:{port}" + BaseRestController.BASE_SOCKET_URL + "/join-room-endpoint";
-    private static final String SUBSCRIBE_ENDPOINT = BaseRestController.BASE_SOCKET_URL + "/%s/subscribe-user";
+    private static final String USER_SUBSCRIBE_ENDPOINT = BaseRestController.BASE_SOCKET_URL + "/%s/subscribe-user";
+    private static final String NOTIFICATION_SUBSCRIBE_ENDPOINT = BaseRestController.BASE_SOCKET_URL + "/%s/subscribe-notification";
 
-    private BlockingQueue<GameDto> blockingQueue;
+    private BlockingQueue<GameDto> userBlockingQueue;
+    private BlockingQueue<GameNotificationDto> notificationBlockingQueue;
     private String baseRestEndpoint;
     private RoomDto room;
     private StompSession hostSession;
 
-    // Predefine problem attributes.
-    private static final String NAME = "Sort a List";
-    private static final String DESCRIPTION = "Sort the given list in O(n log n) time.";
-
     // Predefine user and room attributes.
     private static final String NICKNAME = "rocket";
     private static final String USER_ID = "012345";
     private static final String NICKNAME_2 = "rocketrocket";
     private static final String USER_ID_2 = "098765";
-    private static final String INPUT = "[1, 8, 2]";
-    private static final String OUTPUT = "[1, 2, 8]";
-    private static final long DURATION = 5;
+    private static final String TIME_LEFT = "are ten seconds";
+    private static final long DURATION = 15;
```

Comment:
Note for future: If we continue expanding this file with more tests, we might want to consider testing the time ran out socket update in a separate PR (that way we don't have to wait for multiple seconds/minutes)

Reply:
Makes sense, sounds good 👍